From 551ba7e1e0af3ae7a19a6bf6909db8178a2563cd Mon Sep 17 00:00:00 2001
From: Stefan Hansson <newbyte@postmarketos.org>
Date: Sat, 17 Aug 2024 20:27:40 +0200
Subject: [PATCH] Remove bc suffix from version

abuild doesn't like this. While it might seem appealing to strip this
using some sed invocation to make this more futureproof, in practice
this is a fool's errand as since this release they changed the build
system from Autotools to CMake, so any future release will use CMake
anyway which consequently makes any changes to configure.ac pointless.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index cbd57c4d..59fa82cc 100644
--- a/configure.ac
+++ b/configure.ac
@@ -11,7 +11,7 @@ dnl information on the package
 dnl ---------------------------
 
 dnl update both the version for AC_INIT and the LIBSOFIA_SIP_UA_MAJOR_MINOR
-AC_INIT([sofia-sip], [1.13.45bc])
+AC_INIT([sofia-sip], [1.13.45])
 AC_CONFIG_SRCDIR([libsofia-sip-ua/sip/sofia-sip/sip.h])
 AC_CONFIG_MACRO_DIR([m4])
 AC_SUBST(VER_LIBSOFIA_SIP_UA_MAJOR_MINOR, [1.13])
-- 
2.45.2

